integrationTests/relayers/slowTests/framework/multiversxHandler.go

```diff
@@ -1068,6 +1117,49 @@ func (handler *MultiversxHandler) withdrawFees(ctx context.Context,
 			withdrawFunction, initialBalanceStr, finalBalanceStr, expectedDelta.String()))
 }
 
+// TestWithdrawFeesShouldFail will try to withdraw the fees for the provided token from the safe contract to the owner
+func (handler *MultiversxHandler) TestWithdrawFeesShouldFail(
+	ctx context.Context,
+	token string,
+	expectedDeltaForAccumulated *big.Int,
+) {
+	handler.withdrawFeesShouldFail(ctx, token, expectedDeltaForAccumulated, getTransactionFeesFunction, withdrawTransactionFeesFunction)
+}
+
+func (handler *MultiversxHandler) withdrawFeesShouldFail(ctx context.Context,
+	token string,
+	expectedDelta *big.Int,
+	getFunction string,
+	withdrawFunction string,
+) {
+	queryParams := []string{
```

Comment:
maybe extract L1135-L1156 in a function and use it on L1082 to remove the duplicated code?